tests/test_core.py

```diff
@@ -332,10 +332,14 @@ def test_record_str(daa: Array) -> None:
 
 
 def test_record_to_delayed(daa: Array) -> None:
+    dask.config.set({"awkward.raise-failed-meta": True})
+    dask.config.set({"awkward.optimization.on-fail": "raise"})
```

Comment:
setting the dask config like this is global; should we use the context manager option?:
```python
def test():
    with dask.config.set(
        {
            "awkward.raise-failed-meta": True,
            "awkward.optimization.on-fail": "raise",
        }
    ):
        # ... rest of test
```

Reply:
In fact, this was just for local testing ... I should remove this.